devices/IFrameTransformToIWear/src/IFrameTransformToIWear.cpp

```diff
@@ -124,10 +124,12 @@ class IFrameTransformToIWear::Impl::PoseSensor : public wearable::sensor::IPoseS
     // ------------------------
     PoseSensor(
         IFrameTransformHandler* frameTransformHandler,
+        std::recursive_mutex* mutex,
```

Comment:
Is there a way to avoid passing raw pointers? Maybe using references? In other sensors implementation you saw, we were passing the raw pointer of the pImpl object which is fine because 1) it is always alive and 2) originally it is a unique pointer that cannot be copied.

Reply:
I have done the change you suggested in https://github.com/robotology/wearables/pull/126/commits/62947e2b557f99384dbe5a3ec2e27ab0afb4f7f7, thanks. Let me know if it is ok, and if you have further comment.

Reply:
It looks good, thanks!